src/legacy/core_plugins/expressions/public/np_ready/public/expression_renderer.tsx

```diff
@@ -70,5 +116,25 @@ export const createRenderer = (loader: IExpressionLoader): ExpressionRenderer =>
     mountpoint.current,
   ]);
 
-  return <div {...dataAttrs} className={className} ref={mountpoint} />;
+  return (
+    <div
+      {...dataAttrs}
+      style={{
+        display: state.hasError ? 'none' : 'flex',
```

Comment:
Changing from none to flex seems as if it might introduce some strange behaviors.

Also, I wonder if we should make the overflow `auto` in the case of an error state or something, just in case.

Reply:
This whole style block is causing issues in the Suggestions panel render. What is it currently trying to fix and why are they inline?